packages/react/src/patterns/sections/LeadSpace/__stories__/LeadSpace.stories.js

```diff
@@ -93,10 +93,10 @@ if (DDS_LEADSPACE) {
       };
 
       const images = {
-        mobile: 'https://picsum.photos/id/1076/320/370',
-        tablet: 'https://picsum.photos/id/1076/672/400',
-        default: 'https://picsum.photos/id/1076/1056/480',
-        alt: 'lead space image',
+        mobile: 'https://dummyimage.com/320x370/ee5396/161616',
+        tablet: 'https://dummyimage.com/672x400/ee5396/161616',
+        default: 'https://dummyimage.com/1056x480/ee5396/161616',
+        alt: 'Image alt text',
```

Comment:
Do these need an aspect ratio?

Reply:
I left these out intentionally because I don't know the rules for aspect ratio in the leadspace and didn't want to force anything. So I kept the image dimensions that were being used. @annawen1 @wonilsuhibm, is there an aspect ratio for leadspace images?